internal/k8s/jsonpath/value.go

```diff
@@ -0,0 +1,34 @@
+package jsonpath
+
+import "reflect"
+
+// A hack over reflect.Value that allows maps keys to be settable.
```

Comment:
I first read this as "lets you change the keys in a map" (like, change `{"foo": true}` to `{"bar": true}`), which I don't think is what it's doing - would "allows map values to be settable" or "allows values indexed by maps' keys to be settable" work?

Reply:
good point, clarified the language here a bit